Gems/RecastNavigation/gem.json

```diff
@@ -0,0 +1,21 @@
+{
+    "gem_name": "RecastNavigation",
+    "display_name": "RecastNavigation",
+    "license": "Apache-2.0 Or MIT and zlib license for Recast library",
+    "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
+    "origin": "Open 3D Engine - o3de.org",
+    "origin_url": "https://github.com/o3de/o3de",
+    "type": "Code",
+    "summary": "Recast Navigation gem provides navigation features using Recast/Detour library that can be found at https://github.com/recastnavigation/recastnavigation",
+    "canonical_tags": [
+        "Gem"
+    ],
+    "user_tags": [
+        "Navigation"
+    ],
+    "icon_path": "preview.png",
+    "requirements": "This gem requires https://github.com/recastnavigation/recastnavigation",
```

Comment:
How are those requirements enforced? This reads that you should have this locally.

Reply:
Added a clarification.